baseframe/__init__.py

```diff
@@ -321,6 +323,19 @@ def get_timezone():
     return current_app.config.get('tz') or UTC
 
 
+def get_countries_list():
+    # Returns a localized list of country names and the ISO3166-2 code
+    return get_localized_countries(get_locale())
+
+
+@cache.memoize(timeout=86400)
+def get_localized_countries(locale):
+    pycountry_locale = gettext.translation('iso3166-2', pycountry.LOCALES_DIR, languages=[locale])
```

Comment:
Shouldn't this be `3166-1`? `-2` is for states.

Reply:
I'm having trouble with this. pycountry only seems to be generating locales for `-2`. I'm tryinh to fix that.